demos/signal-protocol/Makefile

```diff
@@ -1,6 +1,8 @@
 LIBSIGNAL_DIR=libsignal-protocol-c
 LIBSIGNAL_BUILD_DIR=$(LIBSIGNAL_DIR)/build
-LLVM_LINK_NAME=$(shell dirname $(shell readlink -f $(shell which clang)))/llvm-link
```

Comment:
`wllvm` [requires an environment variable named `LLVM_LINK_NAME`](https://github.com/travitch/whole-program-llvm/tree/7d1380692a0476daa766f21ea5664d04006cf98d#usage) to function correctly. Renaming it to `LLVM_LINK` could work, but I think you'd need to use `LLVM_LINK_NAME=$(LLVM_LINK)` wherever `wllvm` is being invoked.

Reply:
Thanks! I'm not sure how long that would have taken me to realize.

I think I'm slowly discovering all of the ways to break these tests xD